odoo/addons/base/models/ir_ui_view.py

```diff
@@ -2470,37 +2470,35 @@ def distribute_branding(self, e, branding=None, parent_xpath='',
         if not e.get('data-oe-model'):
             return
 
-        if {'t-raw', 't-out'}.intersection(e.attrib):
+        if e.get('t-out'):
             # nodes which fully generate their content and have no reason to
             # be branded because they can not sensibly be edited
             self._pop_view_branding(e)
         elif self._contains_branded(e):
             # if a branded element contains branded elements distribute own
-            # branding to children unless it's t-raw, then just remove branding
-            # on current element
+            # branding to children, then just remove branding on current element
             distributed_branding = self._pop_view_branding(e)
 
-            if 't-raw' not in e.attrib:
-                # TODO: collections.Counter if remove p2.6 compat
-                # running index by tag type, for XPath query generation
-                indexes = collections.defaultdict(lambda: 0)
-                for child in e.iterchildren(etree.Element, etree.ProcessingInstruction):
-                    if child.get('data-oe-xpath'):
-                        # injected by view inheritance, skip otherwise
-                        # generated xpath is incorrect
-                        self.distribute_branding(child)
-                    elif child.tag is etree.ProcessingInstruction:
-                        # If a node is known to have been replaced during
-                        # applying an inheritance, increment its index to
-                        # compute an accurate xpath for subsequent nodes
-                        if child.target == 'apply-inheritance-specs-node-removal':
-                            indexes[child.text] += 1
-                            e.remove(child)
-                    else:
-                        indexes[child.tag] += 1
-                        self.distribute_branding(
-                            child, distributed_branding,
-                            parent_xpath=node_path, index_map=indexes)
+            # TODO: collections.Counter if remove p2.6 compat
+            # running index by tag type, for XPath query generation
+            indexes = collections.defaultdict(lambda: 0)
```

Comment:
Can you do this todo? (python 2 is not supported for a long time)

```suggestion
            # running index by tag type, for XPath query generation
            indexes = collections.Counter()
```